runbotics/runbotics-orchestrator-ui/src/src-landing/components/PostPagination/SwitchPageButton/SwitchPageButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+
+import Image from 'next/image';
+
+import arrowRight from '#public/images/icons/right.svg';
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import styles from './SwitchPageButton.module.scss';
+
+const SWITCH_PAGE_BUTTON_SIZE = 30;
+
+interface SwitchPageButtonProps {
+    direction: 'Previous' | 'Next';
+    currentPage: number;
+    switchPage: (pageNumber: number) => void;
+}
+
+
+const SwitchPageButton: FC<SwitchPageButtonProps> = ({ direction, currentPage, switchPage }) => {
+    const { translate } = useTranslations();
+    
+    return (
+        <button
```

Comment:
it should be a link + remove switchPage prop

Reply:
use title property for next and specific pages

Reply:
done